### ISSUE TITLE
`detail` command doesn't work if geoweaver server is not started

### ISSUE THREAD

**Issue by ZihengSun** [ESIPFed/pygeoweaver#47]
```
gw detail code
Usage: gw detail code [OPTIONS] PROCESS_ID
Try 'gw detail code --help' for help.

Error: Missing argument 'PROCESS_ID'.
(base) [zsun@hopper1 ~]$ gw detail code  599uqc
Traceback (most recent call last):
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/urllib3/util/connection.py", line 95, in create_connection
    raise err
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/urllib3/util/connection.py", line 85, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 398, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/urllib3/connection.py", line 239, in request
    super(HTTPConnection, self).request(method, url, body=body, headers=headers)
  File "/home/zsun/anaconda3/lib/python3.8/http/client.py", line 1255, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/home/zsun/anaconda3/lib/python3.8/http/client.py", line 1301, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/home/zsun/anaconda3/lib/python3.8/http/client.py", line 1250, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/home/zsun/anaconda3/lib/python3.8/http/client.py", line 1010, in _send_output
    self.send(msg)
  File "/home/zsun/anaconda3/lib/python3.8/http/client.py", line 950, in send
    self.connect()
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/urllib3/connection.py", line 205, in connect
    conn = self._new_conn()
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x7f3cc3f3f160>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/requests/adapters.py", line 440, in send
    resp = conn.urlopen(
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=8071): Max retries exceeded with url: /Geoweaver/web/detail (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3cc3f3f160>: Failed to establish a new connection: [Errno 111] Connection refused'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/zsun/anaconda3/bin/gw", line 8, in <module>
    sys.exit(geoweaver())
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/pygeoweaver/__main__.py", line 180, in get_process_code_command
    get_process_code(process_id)
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/pygeoweaver/commands/pgw_detail.py", line 115, in get_process_code
    r = requests.post(
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/requests/api.py", line 117, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/home/zsun/anaconda3/lib/python3.8/site-packages/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=8071): Max retries exceeded with url: /Geoweaver/web/detail (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3cc3f3f160>: Failed to establish a new connection: [Errno 111] Connection refused'))

````

**Comment by ZihengSun** [comment]
We expect the geoweaver server automatically started if it is not.

**Comment by prasoona14** [comment]
The issue is resolved under PR [#48](https://github.com/ESIPFed/pygeoweaver/pull/48). Let me know if this looks good or if you have any feedback!
